curiouslearning/FeedTheMonsterJS
# fix: CR-174 content changes for Oromo

# Changes
- lang/oromo/ftm_oromo.json

# How to test
- When I play the development APK and open Oromo,
Then I should see the title as “Hiriyyaa kee soori”.

Ref: [CR-174](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-174)


[CR-174]: https://curiouslearning.atlassian.net/browse/CR-174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
